src/reactors/PriorityOrderReactor.sol

```diff
@@ -77,5 +101,13 @@ contract PriorityOrderReactor is BaseReactor {
                 }
             }
         }
+
+        (bytes32 r, bytes32 s) = abi.decode(order.cosignature, (bytes32, bytes32));
```

Comment:
don't you need to skip the cosigner check somehow if past the user's signed start block?

Reply:
ah yes great point - I refactored a little bit to only do all of the cosigner logic if we are before the user's signed start block